src/lib/support/jsontlv/TlvToJson.cpp

```diff
@@ -35,6 +36,9 @@ namespace {
 // and this value is never stored.
 constexpr uint32_t kTemporaryImplicitProfileId = 0xFF01;
 
+// Add the bytes for tag(1:control + 8:tag + 8:length) and structure(1:struct + 1:close container)
```

Comment:
When would we have 8 bytes for tag and 8 bytes for length?  I don't think that ever happens in practice with this encoder.... at least unless the incoming data is > 4GB, and even then the tag would not be 8 bytes.

Reply:
I think it should be at most 2 bytes tag if considering implicit profileId + at most 4 bytes for length if considering the date is bytestring or utf8